src/components/GrantRequest/GrantRequestSection.css

```diff
@@ -84,11 +84,13 @@
 .GrantRequestSection__Field--row {
```

Comment:
If 'row' is not a modifier to the CSS class we should call it `__FieldRow`, right?